docs/development.md

```diff
@@ -12,21 +12,29 @@ Interested in contributing to Masonry? Awesome! Take a look at our [contribution
     go get github.com/opencontrol/compliance-masonry
     ```
 
-1. Install the dependencies.
+1. Run the tool.
 
     ```sh
-    cd $GOPATH/src/github.com/opencontrol/compliance-masonry
-    dep ensure
-    go install .
+    compliance-masonry
     ```
 
-1. Run the tool.
+This should print out usage documentation.
+
+## Updating dependencies
+
+1. Get the `vndr` handling tool.
 
     ```sh
-    compliance-masonry
+    go get github.com/LK4D4/vndr
     ```
 
-This should print out usage documentation.
+1. Update dependencies by running the `vndr` tool in the project.
+
+   ```sh
+   vndr
```

Comment:
Output shows some of these packages are unused & consider removing.

`````
$ vndr
2018/05/30 14:25:13 Collecting initial packages
2018/05/30 14:25:13 Download dependencies
2018/05/30 14:25:17 Starting whole vndr cycle because no package specified
2018/05/30 14:25:17 	Clone golang.org/x/text, revision master
2018/05/30 14:25:17 	Clone github.com/metaleap/go-util, revision master
2018/05/30 14:25:17 	Clone github.com/blang/semver, revision master
2018/05/30 14:25:17 	Clone github.com/onsi/ginkgo, revision master
2018/05/30 14:25:17 	Clone gopkg.in/yaml.v2, revision v2
2018/05/30 14:25:17 	Clone github.com/codegangsta/cli, revision master
2018/05/30 14:25:17 	Clone gopkg.in/fatih/set.v0, revision master
2018/05/30 14:25:17 	Clone vbom.ml/util, revision master
2018/05/30 14:25:17 	Clone github.com/stretchr/testify, revision master
2018/05/30 14:25:17 	Clone github.com/pmezard/go-difflib, revision master
2018/05/30 14:25:17 	Clone github.com/onsi/gomega, revision master
2018/05/30 14:25:17 	Clone github.com/davecgh/go-spew, revision master
2018/05/30 14:25:17 	Clone golang.org/x/net, revision master
2018/05/30 14:25:17 	Clone github.com/stretchr/objx, revision master
2018/05/30 14:25:17 	Clone github.com/Masterminds/vcs, revision master
2018/05/30 14:25:17 	Clone github.com/tg/gosortmap, revision master
2018/05/30 14:25:18 	Finished clone github.com/blang/semver
2018/05/30 14:25:18 	Clone github.com/vektra/errors, revision master
2018/05/30 14:25:18 	Finished clone vbom.ml/util
2018/05/30 14:25:18 	Clone github.com/go-forks/fsnotify, revision master
2018/05/30 14:25:18 	Finished clone github.com/pmezard/go-difflib
2018/05/30 14:25:18 	Finished clone github.com/tg/gosortmap
2018/05/30 14:25:18 	Finished clone github.com/davecgh/go-spew
2018/05/30 14:25:18 	Finished clone github.com/Masterminds/vcs
2018/05/30 14:25:18 	Finished clone github.com/stretchr/objx
2018/05/30 14:25:18 	Finished clone gopkg.in/fatih/set.v0
2018/05/30 14:25:18 	Finished clone github.com/stretchr/testify
2018/05/30 14:25:18 	Finished clone github.com/metaleap/go-util
2018/05/30 14:25:19 	Finished clone github.com/codegangsta/cli
2018/05/30 14:25:19 	Finished clone gopkg.in/yaml.v2
2018/05/30 14:25:19 	Finished clone github.com/onsi/gomega
2018/05/30 14:25:19 	Finished clone github.com/go-forks/fsnotify
2018/05/30 14:25:19 	Finished clone github.com/vektra/errors
2018/05/30 14:25:19 	Finished clone github.com/onsi/ginkgo
2018/05/30 14:25:20 	Finished clone golang.org/x/text
2018/05/30 14:25:21 	Finished clone golang.org/x/net
2018/05/30 14:25:21 Dependencies downloaded. Download time: 3.299376195s
2018/05/30 14:25:21 Collecting all dependencies
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/stretchr/testify/mock
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/stretchr/testify/assert
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/Masterminds/vcs
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo/extensions/table
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/gomega
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/gomega/internal/assertion
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/gomega/internal/asyncassertion
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/gomega/internal/testingtsupport
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/gomega/matchers
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/gomega/types
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/gomega/format
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/gomega/internal/oraclematcher
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/gomega/matchers/support/goraph/bipartitegraph
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/net/html/charset
2018/05/30 14:25:21 WARNING: dependency is not vendored: gopkg.in/yaml.v2
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/net/html
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/text/encoding
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/text/encoding/charmap
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/text/encoding/htmlindex
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/text/transform
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/text/encoding/internal/identifier
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/text/encoding/japanese
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/text/encoding/korean
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/text/encoding/simplifiedchinese
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/text/encoding/traditionalchinese
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/text/encoding/unicode
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/text/language
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/text/internal/language
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/text/internal/language/compact
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/text/internal/tag
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/text/encoding/internal
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/text/internal/utf8internal
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/text/runes
2018/05/30 14:25:21 WARNING: dependency is not vendored: golang.org/x/net/html/atom
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/gomega/matchers/support/goraph/edge
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/gomega/matchers/support/goraph/node
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/gomega/matchers/support/goraph/util
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo/config
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo/internal/codelocation
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo/internal/failer
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo/internal/remote
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo/internal/suite
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo/internal/testingtproxy
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo/internal/writer
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo/reporters
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo/reporters/stenographer
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo/types
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo/reporters/stenographer/support/go-colorable
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo/reporters/stenographer/support/go-isatty
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo/internal/containernode
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo/internal/leafnodes
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo/internal/spec
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo/internal/spec_iterator
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/onsi/ginkgo/internal/specrunner
2018/05/30 14:25:21 WARNING: dependency is not vendored: github.com/vektra/errors
2018/05/30 14:25:21 WARNING: dependency is not vendored: gopkg.in/fatih/set.v0
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/lib/common
2018/05/30 14:25:22 WARNING: dependency is not vendored: vbom.ml/util/sortorder
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/blang/semver
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/lib/standards/versions/1_0_0
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/tools/constants
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/lib/opencontrol/versions/1.0.0
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/lib/opencontrol
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/lib/opencontrol/mocks
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/lib/components/versions/2_0_0
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/lib/components/versions/3_0_0
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/lib/components/versions/3_1_0
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/tools/fs
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/lib/components
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/lib/certifications/versions/1_0_0
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/lib/certifications
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/codegangsta/cli
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/lib/result
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/lib/standards
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/lib/common/mocks
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/lib
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/onsi/gomega/gbytes
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/onsi/gomega/gexec
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/tools/mapset
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/tools/vcs
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/commands/get/resources/mocks
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/tools/fs/mocks
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/tools/vcs/mocks
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/commands/get/resources
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/commands/docs/gitbook
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/tools/certifications
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/commands/docs
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/commands/diff
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/opencontrol/compliance-masonry/commands/get
2018/05/30 14:25:22 WARNING: dependency is not vendored: github.com/tg/gosortmap
2018/05/30 14:25:22 Clean vendor dir from unused packages
2018/05/30 14:25:22 WARNING: package gopkg.in/fatih/set.v0 is unused, consider removing it from vendor.conf
2018/05/30 14:25:22 WARNING: package github.com/stretchr/testify is unused, consider removing it from vendor.conf
2018/05/30 14:25:22 WARNING: package github.com/stretchr/objx is unused, consider removing it from vendor.conf
2018/05/30 14:25:22 WARNING: package github.com/davecgh/go-spew is unused, consider removing it from vendor.conf
2018/05/30 14:25:22 WARNING: package github.com/pmezard/go-difflib is unused, consider removing it from vendor.conf
2018/05/30 14:25:22 WARNING: package github.com/Masterminds/vcs is unused, consider removing it from vendor.conf
2018/05/30 14:25:22 WARNING: package github.com/vektra/errors is unused, consider removing it from vendor.conf
2018/05/30 14:25:22 WARNING: package github.com/go-forks/fsnotify is unused, consider removing it from vendor.conf
2018/05/30 14:25:22 WARNING: package github.com/metaleap/go-util is unused, consider removing it from vendor.conf
2018/05/30 14:25:22 WARNING: package github.com/codegangsta/cli is unused, consider removing it from vendor.conf
2018/05/30 14:25:22 WARNING: package vbom.ml/util is unused, consider removing it from vendor.conf
2018/05/30 14:25:22 WARNING: package gopkg.in/yaml.v2 is unused, consider removing it from vendor.conf
2018/05/30 14:25:22 WARNING: package github.com/blang/semver is unused, consider removing it from vendor.conf
2018/05/30 14:25:22 WARNING: package github.com/tg/gosortmap is unused, consider removing it from vendor.conf
2018/05/30 14:25:22 WARNING: package github.com/onsi/ginkgo is unused, consider removing it from vendor.conf
2018/05/30 14:25:22 WARNING: package github.com/onsi/gomega is unused, consider removing it from vendor.conf
2018/05/30 14:25:22 WARNING: package golang.org/x/net is unused, consider removing it from vendor.conf
2018/05/30 14:25:22 WARNING: package golang.org/x/text is unused, consider removing it from vendor.conf
2018/05/30 14:25:22 Success
2018/05/30 14:25:22 Running time: 9.663889921s
`````